src/multivariate/multinomial.jl

```diff
@@ -25,7 +25,7 @@ struct Multinomial{T<:Real, TV<:AbstractVector{T}} <: DiscreteMultivariateDistri
     Multinomial{T, TV}(n::Int, p::TV) where {T <: Real, TV <: AbstractVector{T}} = new{T, TV}(n, p)
 end
 
-function Multinomial(n::Integer, p::TV; check_args=true) where {T <: Real, TV <: AbstractVector{T}}
+function Multinomial(n::Integer, p::AbstractVector{T}; check_args::Bool=true) where {T<:Real}
```

Comment:
As a reviewer, I'm not really a fan of these kinds of changes in an already massive PR.

Reply:
Which breaking changes? There really shouldn't be any. Here I just removed an unnecessary static type parameter since the line was changed anyway, following the [Julia style guide](https://docs.julialang.org/en/v1/manual/style-guide/index.html#Don't-use-unnecessary-static-parameters). I can move this change  to a separate PR but this seemed a bit unnecessary (and I only noticed this while working on the `check_args` changes, I didn't just include additional changes that I intended to commit).